src/common/locations.ts

```diff
@@ -124,17 +124,15 @@ export function getLocationNameForFips(fips: string): string {
   }
 }
 
-export function getLocationUrlForFips(fips: string): string {
-  // TODO(pablo): Make sure that these urls are uniform, here, the state
-  // URL ends on / but not the county one
+export function getRelativeUrlForFips(fips: string): string {
```

Comment:
In order to construct the /share/xyz/?redirectTo=/us/<state>/explore/xyz URLs I had to change this to be relative instead of the full URL.

I also fixed your TODO.